core/storage/classifier/gae_models_test.py

```diff
@@ -313,6 +337,42 @@ def test_get_deletion_policy(self) -> None:
             .get_deletion_policy(),
             base_models.DELETION_POLICY.NOT_APPLICABLE)
 
+    def test_get_export_policy(self) -> None:
+        sample_dict = base_models.BaseModel.get_export_policy()
+        sample_dict.update(
```

Comment:
Please see my notes on formatting and other comments in your other PRs (for this and above).

Reply:
Will do!